include/openssl/obj_mac.h

```diff
@@ -4301,7 +4301,7 @@
 
 #define SN_id_tc26_wrap_gostr3412_2015_kuznyechik_kexp15                "id-tc26-wrap-gostr3412-2015-kuznyechik-kexp15"
 #define NID_id_tc26_wrap_gostr3412_2015_kuznyechik_kexp15               1183
-#define OBJ_id_tc26_wrap_gostr3412_2015_kuznyechik_kexp15               OBJ_id_tc26_wrap_gostr3412_2015_magma,1L
+#define OBJ_id_tc26_wrap_gostr3412_2015_kuznyechik_kexp15               OBJ_id_tc26_wrap_gostr3412_2015_kuznyechik,1L
```

Comment:
This file is auto-generated. You actually need to edit objects.txt and then run "make update":

https://github.com/openssl/openssl/blob/db42bb440e76399b89fc8ae04644441a2a5f6821/crypto/objects/objects.txt#L1370